spec.html

```diff
@@ -16079,6 +15984,40 @@
           1. Return ? ResolveBinding(_bindingId_).
         </emu-alg>
       </emu-clause>
+
+      <!-- es6num="9.1.11" -->
+      <emu-clause id="sec-enumerate-object-properties" aoid="EnumerateObjectProperties">
+        <h1>EnumerateObjectProperties (_O_)</h1>
+        <p>When the abstract operation EnumerateObjectProperties is called with argument _O_, the following steps are taken:</p>
+        <emu-alg>
+          1. Assert: Type(_O_) is Object.
+          1. Return an Iterator object (<emu-xref href="#sec-iterator-interface"></emu-xref>) whose `next` method iterates over all the String-valued keys of enumerable properties of _O_. The Iterator object must inherit from %IteratorPrototype% (<emu-xref href="#sec-%iteratorprototype%-object"></emu-xref>). The mechanics and order of enumerating the properties is not specified but must conform to the rules specified below.
```

Comment:
And actually the requirement that it MUST inherit from %IteratorPrototype% is kind of bogus since the intention is this isn't observable. Can I simply remove that line?